modules/memo/src/main/ExpireSetMemo.scala

```diff
@@ -8,7 +8,7 @@ final class ExpireSetMemo[K](ttl: FiniteDuration):
     .expireAfterWrite(ttl)
     .build[K, Boolean]()
 
-  def get(key: K): Boolean = cache.underlying.getIfPresent(key) == true
+  def get(key: K): Boolean = cache.underlying.getIfPresent(key)
```

Comment:
`getIfPresent` is a java API that can return null. the `true` check eliminates the `null`s

Reply:
oh, Java 😢